common/technology/technologies/20_military.txt

```diff
@@ -398,6 +401,7 @@ hydraulic_cranes = {
 	
 	modifier = {
 		state_building_port_max_level_add = 3
+		country_convoys_capacity_mult = 0.05
```

Comment:
Increased efficiency of a port's effective convoy production. Port throughput would still require more ships to draw from for the convoys, which would not really increase convoy count efficiently. Higher convoy counts are necessary for late game trade routes to expand more, and this will stimulate that.